gr-digital/lib/qa_header_format.cc

```diff
@@ -29,25 +29,40 @@
 #include <gnuradio/digital/header_format_default.h>
 #include <gnuradio/blocks/unpack_k_bits.h>
 #include <gnuradio/expj.h>
+#include <gnuradio/xoroshiro128p.h>
 #include <volk/volk.h>
 #include <boost/test/unit_test.hpp>
 #include <stdio.h>
 #include <cmath>
 
+static void xoroshiro_fill_buffer(uint8_t *buffer, unsigned int length, uint64_t seed = 42) {
+  uint64_t rng_state[2];
+  xoroshiro128p_seed(rng_state, seed);
+  uint64_t *data_ptr = reinterpret_cast<uint64_t*>(buffer);
+  for(unsigned int i = 0; i < length / 8; ++i) {
```

Comment:
I would have done `sizeof(uint64_t)` instead of 8, but that's no reason not to merge.